docs/src/api/websocket/_signatureSubscribe.mdx

```diff
@@ -54,8 +54,8 @@ Configuration object containing the following fields:
 
 <Field name="enableReceivedNotification" type="bool" optional={true}>
 
-Whether or not to recieve notifications when signatures are recieved by the RPC,
-in addition to when they are processed.
+Whether or not to subscribe for notifications when signatures are received by
```

Comment:
```suggestion
Whether or not to subscribe for notifications when signatures are received by
```

Reply:
was this suggestion supposed to have a change on it?
GH is showing me that no actual change was made on the suggestion

Reply:
You got it I think! Just the spelling of receive.

Reply:
Ahhh I see it now! Nice catch. Strange my spell check did not catch it, and GH diffs do not actually highlight it since it is in the same word...